netebpfext/net_ebpf_ext.h

```diff
@@ -236,19 +236,17 @@ typedef struct _net_ebpf_extension_flow_context_parameters
 
 typedef enum _net_ebpf_extension_hook_id
 {
-    EBPF_HOOK_OUTBOUND_L2 = 0,
-    EBPF_HOOK_INBOUND_L2,
-    EBPF_HOOK_ALE_RESOURCE_ALLOC_V4,
+    EBPF_HOOK_ALE_RESOURCE_ALLOC_V4 = 0,
```

Comment:
Is this numbering change a breaking change?   It's potentially ok if so since we're making other breaking changes, but this would need to go in the list of breaking changes.  Otherwise, we could preserve the original numbers and start with `= 2`.

Reply:
Yes, this is a breaking change. The code accesses other structures by index and we cannot maintain the same values.  I tried with starting value of 2 but ran into several issues.